### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by franciscomorais**
This PR adds the initial version of `goflow`.

### CODE REVIEW

File: src/commands/install.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+/**
+ * Export `install` command.
+ */
+
+export default async function(library, options) {
+  const { result, splitLibrary } = await checkLibrary(library);
+
+  if (!splitLibrary) {
+    messages.libraryNotFound();
```
Comment (franciscomorais) [inline src/commands/install.js]: pass `result` as a parameter.

File: src/commands/install.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+    return;
+  }
+
+  if (splitLibrary.length !== 2) {
+    messages.cantReadVersion();
```
Comment (franciscomorais) [inline src/commands/install.js]: same as above.

File: src/commands/install.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    } : {};
+  }
+};
+
+/**
+ * Export `install` command.
+ */
+
+export default async function(library, options) {
+  const { result, splitLibrary } = await checkLibrary(library);
```
Comment (bsonntag) [inline src/commands/install.js]: `checkLibrary` can return undefined, which will make this destructuring throw.

File: src/commands/install.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+/**
+ * Messages.
+ */
+
+const messages = {
+  cantReadVersion: library => console.log(`\nERR: Something wrong. I can't read name and version of ${result}\n`),
+  libraryNotFound: library => console.log(`\nERR: ${library} not found. Are you sure this library is installed?\n`),
+  somethingWrong: library => console.log(`\nSomething wrong on try install ${library} flow dependency`),
+  successful: library => console.log(`\nThe library ${library} was installed sucessful.\n`)
```
Comment (bsonntag) [inline src/commands/install.js]: You have a typo here. This should say "installed sucessfully".

File: src/commands/install.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+import exec from 'async-exec';
+
+/**
+ * Messages.
+ */
+
+const messages = {
+  cantReadVersion: library => console.log(`\nERR: Something wrong. I can't read name and version of ${result}\n`),
+  libraryNotFound: library => console.log(`\nERR: ${library} not found. Are you sure this library is installed?\n`),
+  somethingWrong: library => console.log(`\nSomething wrong on try install ${library} flow dependency`),
```
Comment (bsonntag) [inline src/commands/install.js]: "Something went wrong while installing"

File: package.json [hunk 6]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "goflow",
+  "version": "1.0.0",
+  "description": "This application help to install flow dependencies using the version installed in project.",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "babel src -d bin",
+    "build-watch": "babel src -d bin --watch",
+    "start": "nodemon --exec babel-node --presets env,stage-2 src/index.js",
```
Comment (bsonntag) [inline package.json]: You already have these presets on you `.babelrc`, so why repeat them here?

File: .babelrc [hunk 7]
```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    "env",
```
Comment (bsonntag) [inline .babelrc]: This is meant to only run on node, so what about setting node as the target of the env preset?

```js
"presets": [
  ["env", {
    "targets": {
      "node": "<version of node you want to support>"
    }
  }],
  "stage-2"
}
```

**Comment by franciscomorais** [comment]
@bsonntag I fix all your comments. Please check the readme.md too. 

Thanks.

**Pull request merged**